### PR TITLE
fix bug UI create book and validation

### PR DESCRIPTION
**Pull request by at-duongtruong**
# Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: #3.1 and #3.2

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

#47

### CODE REVIEW

File: resources/views/backend/books/create.blade.php [hunk 1]
```diff
@@ -87,14 +87,14 @@
...
+                          <label for="InputAuthor">{{ __('books.author') }}</label>
+                          <input type="text" class="form-control" name="author" placeholder="{{ __('books.author') }}" value="{!! old('author') !!}">
                           @if($errors->first('author')) 
                             <span class="text-danger">{{ $errors->first('author') }}</span>
                           @endif
                         </div>
                         <div class="col-xs-6">
-                          <label for="exampleInputYear">{{ __('books.year') }}</label>
+                          <label for="InputYear">{{ __('books.year') }}</label>
                           <input type="number" class="form-control" name="year" placeholder="" value="{!! old('year') !!}">
```
Comment (at-trampham) [inline resources/views/backend/books/create.blade.php]: a thêm `placeholder` cho `year` nữa anh

File: app/Http/Controllers/Admin/BookController.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
 use App\Http\Requests\Backend\EditBookRequest;
-use App\Http\Requests\backend\CreateBookRequest;
+use App\Http\Requests\Backend\CreateBookRequest;
 use Illuminate\Support\Facades\DB;
```
Comment (at-trampham) [inline app/Http/Controllers/Admin/BookController.php]: fix ` use Illuminate\Support\Facades\DB;` thành `use DB;` là được rồi anh

File: app/Http/Requests/Backend/CreateBookRequest.php [hunk 3]
```diff
@@ -25,16 +26,16 @@ public function rules()
...
-            'unit'        => 'in:'.book::TYPE_VND.','
-                                  .book::TYPE_DOLAR.','
-                                  .book::TYPE_EURO.','
-                                  .book::TYPE_YEN.',',
-            'iddonator'   => 'required',
+            'unit'        => 'in:'.Book::TYPE_VND.','
+                                  .Book::TYPE_DOLAR.','
+                                  .Book::TYPE_EURO.','
+                                  .Book::TYPE_YEN.',',
+            'from_person' => 'required|max:10',
```
Comment (at-tuanphan) [inline app/Http/Requests/Backend/CreateBookRequest.php]: max này là max giá trị mà em

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**